src/tasks/barfTurn.ts

```diff
@@ -0,0 +1,6 @@
+import { barfTasks } from "./barfTasks";
+import { runBarfTasks } from "./engine";
+
+export function barfTurns(): void {
+  runBarfTasks(barfTasks);
```

Comment:
I know we have something like this for dailies as well, but just as a file structure thing the existence of this feels kind of silly.